### ISSUE TITLE
Examples are not compatible with Net Core 3.x

### ISSUE THREAD

**Issue by gregoriusus** [iamkinetic/NEventSocket#4]
Solution builds ok on Net Core 2.x. But when changing to Net Core 3.0, example project doesn't build. Get a lot of errors like:
'ILifetimeScope' does not contain a definition for 'ImplementationsFor' and no accessible extension method 'ImplementationsFor' accepting a first argument of type 'ILifetimeScope' could be found 

Guess some extensions are missing in Net Core 3.x.

Any ideas?

Best regards, Gregor

**Comment by iamkinetic** [comment]
Can you post the full exceptions?

**Comment by gregoriusus** [comment]
Hi!

Below are the errors. It looks like something with reference to Autofac. Project is configured for Net Core 3.1

2>------ Rebuild All started: Project: NEventSocket.Examples, Configuration: Debug Any CPU ------
2>NEventSocket Core\NEventSocket.Examples\CommandLineTaskRunner.cs(6,7,6,10): error CS0246: The type or namespace name 'Net' could not be found (are you missing a using directive or an assembly reference?)
2>NEventSocket Core\NEventSocket.Examples\NetCore\CommandLineReader.cs(6,7,6,10): error CS0246: The type or namespace name 'Net' could not be found (are you missing a using directive or an assembly reference?)
2>NEventSocket Core\NEventSocket.Examples\NetCore\CommandLineReader.cs(7,7,7,10): error CS0246: The type or namespace name 'Net' could not be found (are you missing a using directive or an assembly reference?)
2>NEventSocket Core\NEventSocket.Examples\NetCore\CommandLineReader.cs(8,7,8,10): error CS0246: The type or namespace name 'Net' could not be found (are you missing a using directive or an assembly reference?)
2>Done building project "NEventSocket.Examples.csproj" -- FAILED.

**Comment by iamkinetic** [comment]
You're absolutely right, I haven't updated the "NEventSocket.Examples" project.   Honestly I'm not sure I will, but you can use the unit tests as documentation.

**Comment by danbarua** [comment]
@gregoriusus HI, nice to see this project is still alive.

The Examples app is a basic console app that uses AutoFac to spin up the examples.  You could just copy and paste the example code for each one into a vanilla console app and remove un-needed dependencies to try them out.
